README.md

```diff
@@ -1,5 +1,81 @@
-### UFV CSA Website
+# UFV CSA Website
 
-This repository contains the source data for the UFV Computing Student Association's website.
+## Summary
 
-The website is currently in development. Pull requests for content, styling, or design changes are welcome.
+This repository contains the source data for the UFV Computing Student Association's website!
+
+### Goals
+
+This repository has a few key goals:
+
+1. Maintain the UFV CSA website with up-to-date and relevant content for CSA members.
+2. Provide CSA members with an opportunity to work with and learn valuable workplace skills including:
+   - Team Communication: How to effectively work with others on a real-world project.
+   - Git / Github Workflows: How to manage code-bases using git / github
+   - Realworld Web Technologies: How to work with modern web technologies like Gatsby.js, React, and Node.
+
+## Contributing
+
+**The website is currently in development!** If you have any ideas involving:
+
+1. **Content:** Found a typo on the website? Have an even't you'd like to share with other members of the CSA? Have other content ideas? We'd love for you to make a contribution!
+2. **Styling:** Don't like the way the site looks? Found a visual bug? We're open to suggestions and are planning on re-designing the site soon so let us know!
+3. **Use-cases & Design:** Do you have an idea for a useful new feature? Do you feel like a feature on the site could be implemented better? Let's work on it!
+4. **Pretty Much Anything Else**: We're open to general suggestions as well.
+
+If anything listed above sounds like something you'd like to do, **We'd love for you to contribute and become part of the CSA's Open Source Community**.
+
+We've tried to design this repository to be as accessible to all as possible. Whether you've never commited to open source before, or you're an active member of the community
```

Comment:
WIP sentence?

Reply:
Ha, nope must have just got distracted. Thanks for pointing that out.